README.md

```diff
@@ -1,31 +1,62 @@
 [RethinkDB](http://www.rethinkdb.com)
 =================
 
-RethinkDB is an open-source distributed database. It has an intuitive
-query language, automatically parallelized queries, and simple
+RethinkDB is an open-source distributed database. It has an
+intuitive query language, automatically parallelized queries, and simple
 administration.
 
-To get started, check out [rethinkdb.com](http://rethinkdb.com)!
+To learn more, check out [rethinkdb.com](http://rethinkdb.com)!
 
-Quick start
```

Comment:
I think it's really important to add another section before the quick start that explains the motivation for using RethinkDB. I'd give a brief explanation of the kinds of apps users can build (e.g. collaborative web apps, messaging apps, realtime multiplayer games, streaming analytics, connected devices apps), and then give a list of 3-5 blog posts with realtime examples (you can link to the most prominent ones that @segphault built and blogged about on our blog).